minetest/minetest
# Enable chat clickable weblinks by default

Make the ability to click on chat weblinks enabled by default. This does not relate to the roadmap, and is instead a "quality of life" commit.

## How to test
* Build minetest, and don't modify the config file
* Start a game
* Type a weblink into chat
* It should be enabled by default
